### PR TITLE
Update Puppet Providers - Gitter

### PR DESCRIPTION
**Pull request by Soumi7**
Updates Puppet Providers -  Gitter section in references.
 
References issue #890

ScreenShots :

Screenshot from 2021-07-18 12-16-55

Screenshot from 2021-07-18 12-17-03

Screenshot from 2021-07-18 12-17-16

Screenshot from 2021-07-18 12-17-19

Screenshot from 2021-07-18 12-17-22

### CODE REVIEW

**Comment by shraddhavp** [comment]
@Soumi7   please make the CLI green

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by proudofsimin — APPROVED** [review]
Thank you.

**Comment by Rohitesh-Kumar-Jain** [comment]
@Soumi7

**Comment by Rohitesh-Kumar-Jain** [comment]
@shraddhavp

**Comment by Soumi7** [comment]
Have updated. Once the CLI turns green, its ready to merge

**Comment by Soumi7** [comment]
@Rohitesh-Kumar-Jain

File: docusaurus/docs/puppet-providers/gitter.md [hunk 1]
```diff
@@ -63,13 +64,65 @@ npm start
...
+
+To send an image message to your Gitter room, three API calls are required:
+
+1. Generate signature: Call api.gitter.im and get a key to use with transloadit service.
+2. Get transloadit server: Call api2.transloadit.com with the key from step 1, then get a host domain name for using with step 3.
+3. Upload file: Call the server domain name from step 2, and send the image file to it.
+
+Here’s the demo source code for sending an image message to a Gitter room with Wechaty:
+
+```sh
```
Comment (iamrajiv) [inline docusaurus/docs/puppet-providers/gitter.md]: Replace `sh` with `ts`.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: docusaurus/docs/puppet-providers/gitter.md (hunk 1 above)
Comment (Soumi7) [reply docusaurus/docs/puppet-providers/gitter.md]: Done.

**Review by huan — APPROVED** [review]
LGTM.

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Comment by Soumi7** [comment]
@iamrajiv as you requested changes, we will need your approval to merge this PR. Can you please approve this?

**Review by shraddhavp — APPROVED** [review]
LGTM

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**